src/silly_sprites/scene.inl

```diff
@@ -183,8 +183,8 @@ namespace sly {
 
     inline Scene::Scene(AppContext* const app_context) : m_app_context{ app_context } {
         m_registry.on_destroy<Script>().connect<&Scene::on_script_destroyed>(*this);
-        instantiate(Script{ "Player" });
-        instantiate(Script{ "Player" });
+        //instantiate(Script{ "Player" });
+        //instantiate(Script{ "Player" });
```

Comment:
should we leave that as a comment or did you fix it?
i wasn't arround at that time.

Reply:
I think the underlying error has been fixed now. There's still a problem when the scene is destroyed before destroying all active game objects, but it doesn't seem to result in any memory errors. So I'm re-adding those two lines.